bin/util/bibblio_export

```diff
@@ -25,12 +26,24 @@ parser.add_argument(
     '--languages', type=unicode, nargs='*', default=u'eng',
     help='Look for Works in these languages'
 )
+parser.add_argument(
+    '--fiction', action='store_true',
+    help='Export fiction and nonfiction texts'
+)
```

Comment:
My understanding is that you normally only include nonfiction, and this _expands_ it to include fiction. But --fiction sounds like you're doing fiction _instead. Calling it --include-fiction would be more clear IMO.